sync/etcd/etcd.go

```diff
@@ -61,7 +61,7 @@ func (s *Sync) Update(key, jsonString string) error {
 }
 
 //Delete sync update sync
-func (s *Sync) Delete(key string) error {
+func (s *Sync) Delete(key string, prefix bool) error {
 	s.etcdClient.Delete(key, false)
```

Comment:
etcd v2 does not support 'prefix'. It supports 'recursive' because in Etcd v2 has a concept of 'directory'. 'recursive' is similar to 'prefix', but strictly speaking, it is different.

Therefore, actually I considered if this prefix argument should pass it to etcdClient.Delete() or not, then I decided not to pass. However, I do not have strong opinion to it.

Ok, I will pass this prefix argument to etcdClient.Delete(), and produce some info or warning log like 'Prefix option is translated as Recursive in ETCD v2'.